.github/workflows/nox.yml

```diff
@@ -11,4 +11,4 @@ jobs:
       - uses: wntrblm/nox@2024.04.15
         with:
           python-versions: "3.9"
-      - run: nox -s lint format type_hints smoke_tests unit_tests
+      - run: nox --force-python 3.9 -s lint format type_hints smoke_tests unit_tests
```

Comment:
There was an oversight in the noxfile that allowed some of the tests to run on arbitrary python versions.  This has been fixed so I'll revert this after merge :+1: